shared/js/media/video_player.js

```diff
@@ -471,6 +483,20 @@ function VideoPlayer(container) {
     }
   });
 
+  slider.addEventListener('keypress', function(e) {
+    // Currently, we only allow screen reader users to adjust sliders with a
+    // constant step size (there is no page up/down equivalent). In the case
+    // of videos, we make sure that the maximum amount of steps for the entire
+    // duration is 20, or 2 second increments if the duration is less then 40
+    // seconds.
+    var step = Math.max(player.duration/20, 2);
+    if (e.keyCode == e.DOM_VK_DOWN) {
+      player.currentTime -= step;
+    } else {
```

Comment:
Please explicitly check for VK_UP here instead of just using else.  In the past, hardware buttons on the device have sent key events and there is talk of exposing those to apps. So if something else starts sending HOME or ESCAPE events, for example, I don't want those trigger this code.